src/transport/tls/cert_resolver.rs

```diff
@@ -108,8 +119,15 @@ impl rustls::ResolvesServerCert for CertResolver {
             return None;
         };
 
+        let cert = if let Some(ref certified_key) = self.certified_key {
+            &certified_key.cert
+        } else {
+            debug!("no certificate yet");
```

Comment:
This log message is redundant with the "no server certificate chain resolved" message from Rustls so I think we should remove it.